iot/api/CloudIotSample/Program.cs

```diff
@@ -1376,6 +1491,209 @@ public static object ClearRegistry(string projectId, string cloudRegion, string
         }
         //[END iot_clear_registry]
 
+
+        //[START iot_attach_device]
+        public static object AttachDevice(MqttClient client, string deviceId)
+        {
+            var attachTopic = $"/devices/{deviceId}/attach";
+            Console.WriteLine("Attaching: {0}", attachTopic);
+            var attachPayLoad = "{}";
```

Comment:
In the other samples, we added the authorization JWT payload that is [passed to the function by the caller](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iot/api-client/mqtt_example/cloudiot_mqtt_example.py#L184).